prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -156,21 +156,20 @@ class WorkflowEngine(
 
     /**
      * Checks if the [sender] has already sent this report by comparing the [digest] against existing records for
-     * this sender in the report_file table. If a duplicate is found an ActionError is thrown which will be picked up
-     * by ReportFunction and logged in action and action_log.
+     * this sender in the report_file table. Return true if a duplicate is found
      */
-    fun verifyNoDuplicateFile(
+    fun isDuplicateFile(
```

Comment:
You are using the term file here again (I know it was used before).  Report?

Reply:
Removed this function entirely.